test/HiGlassComponentTest.js

```diff
@@ -2693,7 +2696,8 @@ describe('Simple HiGlassComponent', () => {
       const horizontalHeatmap = getTrackObjectFromHGC(hgc.instance(), 'aa', 'hh1');
 
       const svg = horizontalHeatmap.exportColorBarSVG();
-      const rect = svg.getElementsByClassName('color-rect')[0];
+      const rects = svg.getElementsByClassName('color-rect');
```

Comment:
Just to check, `rect` is not used down the line right?

Reply:
No: The test has only one more assertion after this point.